src/main.cpp

```diff
@@ -5,22 +5,47 @@
 
 //how to use:
 //./<binary> xpos ypos
+
+
+SDL_Window* window_;
+BackupCamera* backupCamera_;
+bool quitSignal_;
```

Comment:
main is not a class, requiring the variables here be global is a code smell. Generally c++ should require very very few global variables. Some options would be abstracting out this functionality into a class. So main would just be something like `BackupCamera.run()`, and everything in main right now would be into a BackupCamera object.